src/dispensary.js

```diff
@@ -142,26 +142,30 @@ export default class Dispensary {
     var files = [];
 
     for (let version of library.versions) {
-      if (library.filename) {
-        files.push({
-          file: library.filename,
-          fileOut: library.filenameOutput || library.filename,
-          index: index,
-          library: library,
-          version: version,
-          minified: false,
-        });
-      }
+      var minVersion = library.minVersion;
+      if (!minVersion || (minVersion && naturalCompare(
```

Comment:
Can `(minVersion && naturalCompare(minVersion, version) <= 0)` fit on one line instead? That would be easier to read.